package.json

```diff
@@ -255,6 +249,18 @@
 					"default": "lowercase",
 					"description": "Default casing for the completion of items of the ieee and standard libraries."
 				},
+				"VhdlLinter.style.unusedPrefix": {
+					"scope": "window",
+					"type": "string",
+					"default": "",
+					"description": "Prefix for unused objects (no unused warning is shown for these objects)"
+				},
+				"VhdlLinter.style.unusedSuffix": {
+					"scope": "window",
+					"type": "string",
+					"default": "",
+					"description": "Prefix for unused objects (no unused warning is shown for these objects)"
+				},
```

Comment:
We had unused suffix before, can we keep this as a default?

Reply:
I am not sure about default values in general. I would prefer the default settings to be consistent and, therefore, I removed the unused suffix. But maybe we want to default to a suffix for all styles and then maybe add a hint towards the settings in the namingStyle rule?

Reply:
In general I think the default should interfere with existing code bases as little as possible while providing as much functionality as possible.
So having a default unused suffix/prefix is good I think (somebody randomly using _unused is unlikely).
But probably we should use empty defaults for the remaining styles. Or is there some existing style recommendation similar to pep8 or sth?